src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball ball1 = Lottery.getRandomBall();
+        Ball ball2 = Lottery.getRandomBall();
+        Ball ball3 = Lottery.getRandomBall();
+
+        System.out.println("Ball 1: " + ball1);
+        System.out.println("Ball 2: " + ball2);
+        System.out.println("Ball 3: " + ball3);
```

Comment:
please see the common mistakes:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery